src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendPaperCaseContestedMidHandlerTest.java

```diff
@@ -51,6 +61,36 @@ private static Stream<Arguments> testCanHandle() {
         );
     }
 
+    @Test
+    void testGivenExpressPilotEnabled_ThenExpressCaseServiceCalled() {
+        FinremCallbackRequest callbackRequest = buildCallbackRequest();
+
+        when(featureToggleService.isExpressPilotEnabled()).thenReturn(true);
+
+        FinremCaseData caseData = callbackRequest.getCaseDetails().getData();
+        FinremCaseData caseDataBefore = callbackRequest.getCaseDetailsBefore().getData();
+
+        handler.handle(callbackRequest, AUTH_TOKEN);
+
+        verify(expressCaseService, times(1)).setExpressCaseEnrollmentStatus(caseData);
+        verify(expressCaseService, times(1)).setWhichExpressCaseAmendmentLabelToShow(caseData, caseDataBefore);
```

Comment:
you don't have to put `times(1)` which is already a default value?!

Reply:
@so99y Thanks, news to me too.  Will change and push up for review.